### ISSUE TITLE
[Feature Request] Headless chrome / Heroku integration

### ISSUE THREAD

**Issue by Oplikk** [TychoTheTaco/Twitch-Drops-Bot#110]
Is it possible to make this run with the headless version of chrome, and possibly host this on Heroku?

**Comment by TychoTheTaco** [comment]
Yes you can set `"headless": true` and `headless_login": true` in the config. I'm not familiar with Heroku but I don't see any reason it wouldn't work.

**Comment by Oplikk** [comment]
Could not get the heroku chrome to work, Im too stupid for this. Someone might figure it out
